tests/functional/configs/test_custom_node_colors_configs.py

```diff
@@ -304,7 +305,7 @@ def test__invalid_color_config_block(
         self,
         project,
     ):
-        with pytest.raises(ValidationError):
+        with pytest.raises((ValidationError, ConfigUpdateError)):
```

Comment:
This caused an issue in CI because of the execution order of tests. Since Gerda also [ran into this before](https://github.com/dbt-labs/dbt-core/issues/8311), I am just fixing it here.